test/unit/lua_msgpack.c

```diff
@@ -101,6 +103,22 @@ test_translation_in_encoding(lua_State *L)
 	ibuf_reset(ibuf);
 	mpstream_reset(&stream);
 
+	enum mp_type type;
+	struct tt_uuid uuid;
+	memset(&uuid, 0, sizeof(uuid));
+	luaT_pushuuid(L, &uuid);
+	luamp_encode_with_translation(L, luaL_msgpack_default, &stream, 1,
+				      translation, &type);
+	lua_pop(L, 1);
+	mpstream_flush(&stream);
+	const char *mp_expected = "\xd8\x02\x00\x00\x00\x00\x00\x00\x00"
+				  "\x00\x00\x00\x00\x00\x00\x00\x00\x00";
+	ok(strncmp(ibuf->buf, mp_expected, ibuf_used(ibuf)) == 0,
+	   "UUID is encoded as MP_EXT");
+	ok(type == MP_EXT, "type of UUID is MP_EXT");
+	ibuf_reset(ibuf);
+	mpstream_reset(&stream);
```

Comment:
Please move to a new function (for example, `test_encode_ext`) because this case has nothing to do with translation.

Reply:
Done.